releasenotes/notes/fix-raw-feature-vector-copy-20534d595a0056de.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    This fix removes the `_num_qubits` variable of `RawFeatureVector`, which caused issues on the `copy()` and `bind_parameters()` methods.
```

Comment:
Since `_num_qubits` is a private variable, and the release note is really for user facing changes, where they should not be concerned with/accessing private aspects, can we just simply say something like RawFeatureVector was fixed so now copy and bind_parameter methods work correctly.

Reply:
Updated as @Cryoris suggested below.